templates/docker-virt-manager.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Docker-Virt-Manager</Name>
+  <Repository>mber5/virt-manager:latest</Repository>
+  <Registry>https://hub.docker.com/r/mber5/virt-manager</Registry>
+  <Network>bridge</Network>
+  <Shell>bash</Shell>
+  <Privileged>true</Privileged>
+  <Support>https://forums.unraid.net/topic/77603-virt-manager-intel-gpu-tools-and-more-dockers/</Support>
+  <Project>https://virt-manager.org/</Project>
+  <Overview>GTK Broadway web UI for libvirt. The virt-manager application is a desktop user interface for managing virtual machines through libvirt. It primarily targets KVM VMs, but also manages Xen and LXC (linux containers). It presents a summary view of running domains, their live performance &amp;amp; resource utilization statistics. Wizards enable the creation of new domains, and configuration &amp;amp; adjustment of a domain&#x2019;s resource allocation &amp;amp; virtual hardware. Uses GTK3 Broadway (HTML5) backend--no vnc, xrdp, etc needed! Also has DARK MODE</Overview>
+  <Category>Tools:</Category>
+  <Beta>true</Beta>
+  <WebUI>http://[IP]:[PORT:8185]</WebUI>
+  <TemplateURL>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/UnRaid/template/docker-virt-manager.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/interbiznw/docker-virt-manager/main/virt-manager-icon.png</Icon>
+  <DonateText>If you like my work please consider Donating.</DonateText>
+  <DonateLink>https://interbiznw.github.io/</DonateLink>
+  <Description>GTK Broadway web UI for libvirt. The virt-manager application is a desktop user interface for managing virtual machines through libvirt. It primarily targets KVM VMs, but also manages Xen and LXC (linux containers). It presents a summary view of running domains, their live performance &amp;amp; resource utilization statistics. Wizards enable the creation of new domains, and configuration &amp;amp; adjustment of a domain&#x2019;s resource allocation &amp;amp; virtual hardware. Uses GTK3 Broadway (HTML5) backend--no vnc, xrdp, etc needed! Also has DARK MODE</Description>\
+  <Config Name="KVM" Target="/dev/kvm" Default="" Mode="" Description="Container Path: /dev/kvm" Type="Device" Display="always" Required="false" Mask="false">/dev/kvm</Config>
+  <Config Name="/var/run/libvirt/libvirt-sock" Target="/var/run/libvirt/libvirt-sock" Default="" Mode="rw" Description="Container Path: /var/run/libvirt/libvirt-sock" Type="Path" Display="always" Required="false" Mask="false">/var/run/libvirt/libvirt-sock</Config>
+  <Config Name="/var/lib/libvirt/images" Target="/var/lib/libvirt/images" Default="" Mode="rw" Description="Container Path: /var/lib/libvirt/images" Type="Path" Display="always" Required="false" Mask="false">/var/lib/libvirt/images</Config>
```

Comment:
These looks like they should all be hidden under advanced view.

Reply:
Sorry, new to templates. Will check that out and change.